libcloud/compute/drivers/linode.py

```diff
@@ -997,6 +1038,12 @@ def create_node(
         :keyword ex_private_ip: whether or not to request a private IP
         :type    ex_private_ip: ``bool``
 
+        :keyword ex_userdata: add cloud-config compatible userdata to be
```

Comment:
I think it would actually be better to let user pass in a raw string and then we can take care of base64 encoding the string inside the method ourselves.

This way the method abstracts away this implementation detail from the end user (we already do a similar thing in a bunch of places).

Reply:
@Dorthu Would you like me to make that change? ^^^

Reply:
I agree that it would be more convenient for the library to handle the encoding, especially if there's precedent elsewhere in libcloud, as long as that behavior is featured prominently in the documentation.

Reply:
Will do! Thanks guys.

Reply:
okie dokie @Kami @Dorthu  I have pushed that change (and tested it against Linode).